weblate/machinery/tests.py

```diff
@@ -1278,6 +1279,38 @@ def test_batch(self, machine=None):
             super().test_batch(machine=machine)
 
 
+class IBMTranslationTest(BaseMachineTranslationTest):
+    MACHINE_CLS = IBMTranslation
+    EXPECTED_LEN = 1
+    ENGLISH = "en"
+    SUPPORTED = "zh_Hant"
+    CONFIGURATION = {
+        "url": "https://api.region.language-translator.watson.cloud.ibm.com/"
+        "instances/id",
+        "key": "",
+    }
+
+    def mock_empty(self):
+        raise SkipTest("Not tested")
+
+    def mock_error(self):
+        raise SkipTest("Not tested")
+
+    def mock_response(self):
+        responses.add(
+            responses.GET,
+            "https://api.region.language-translator.watson.cloud.ibm.com/"
+            "instances/id/v3/languages?version=2018-05-01",
+            json={"languages": [{"language": "en"}, {"language": "zh-TW"}]},
+        )
+        responses.add(
+            responses.POST,
+            "https://api.region.language-translator.watson.cloud.ibm.com/"
+            "instances/id/v3/translate?version=2018-05-01",
+            json={"text": ["window"], "source": "en", "target": "zh-TW"},
```

Comment:
According to [the docs](https://cloud.ibm.com/apidocs/language-translator?code=try#translate), the response looks different:

```json
{
  "translations": [
    {
      "translation": "Hola, ¿cómo estás hoy?"
    }
  ],
  "word_count": 7,
  "character_count": 25
}
```

Reply:
```suggestion
            json={"translations": [{      "translation": "window"    }  ],  word_count": 1,  "character_count": 6},
```